docs/identities/native-browser.mdx

```diff
@@ -15,6 +15,8 @@ such as sign in, registration, profile changes, and so on. When using this endpo
 When a user signs in successfully, Ory will issue an Ory Session Cookie. Calling `ory.toSession()` will return the same session
 but does not require any additional calls when used in client-side browser apps (for example React, Vue, Angular).
 
+![Architecture for browser-based app integrated with Ory Identities](./_static/browser.svg)
```

Comment:
The graphic is good, but in this context I find it confusing that we explain how the Ory console works and which services are in Ory Identities. In this context, I want to understand what the difference between native and browser APIs is, and how cookies and tokens work, no?

Reply:
yes, that is a good point. 
Can you think of another place where the graphic would be more fitting? 
Maybe in the [introduction](https://www.ory.sh/docs/welcome)?

Reply:
It would fit in a software architecture document I think. For people who really want to get into it. I don't think it belongs into the introduction which should focus on high level explanations and concepts and be helpful in your first 2 minutes of reading the docs